libs/core/src/lib/combobox/combobox.component.html

```diff
@@ -21,7 +21,10 @@
 </ng-template>
 
 <ng-template let-term="term" #itemSource>
-    <span *ngIf="!itemTemplate" [innerHTML]="term | displayFnPipe: displayFn | highlight: inputText:highlighting">
+    <span
+        *ngIf="!itemTemplate"
+        [innerHTML]="term | displayFnPipe: displayFn | highlight: inputText:highlighting:filterHighlight"
```

Comment:
I believe you can use `inputText : (highlighting && filterHighlight)` . so you can get rid of the additional parameter for pipe